framework/test/singletonframework/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+project(singletonframework)
```

Comment:
I agree. I'll add one within the header or source file. I think that'll be more visible.